### PR TITLE
feature/LP145_text_components

### PR DESCRIPTION
**Pull request by irv205**
image

### CODE REVIEW

File: core/designsystem/src/main/java/com/iteneum/designsystem/components/Text.kt [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+import com.iteneum.designsystem.theme.Boulder
+import com.iteneum.designsystem.theme.LPTypography
+
+/**
+[LPTitleLarge] is a composable function that displays a large title text with the specified label, color, and optional text alignment. It can also be clickable and trigger an onClick event.
+@param modifier Modifier is used to apply custom styles and parameters to the title, such as the width and height of the component.
+@param label The text label to be displayed as the title.
+@param color The color of the title text.
+@param textAlign Optional text alignment parameter, default is TextAlign.Center.
+@param onClick Optional high-order function that defines the click behavior when the title is clicked.
```
Comment (melissaquezada94) [inline core/designsystem/src/main/java/com/iteneum/designsystem/components/Text.kt:18]: I don't see the param onClick in the function, could you remove the param from the documentation if it is not need in the function or add the param onClick in the function if it need

File: core/designsystem/src/main/java/com/iteneum/designsystem/components/Text.kt [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+@author Irving Ulises Gonzalez Uscanga
+
+[LPTitleMedium] is a composable function that displays a medium-sized title text with the specified label, color, and optional text alignment. It can also be clickable and trigger an onClick event.
+@param modifier Modifier is used to apply custom styles and parameters to the title, such as the width and height of the component.
+@param label The text label to be displayed as the title.
+@param color The color of the title text.
+@param textAlign Optional text alignment parameter, default is TextAlign.Center.
+@param onClick Optional high-order function that defines the click behavior when the title is clicked.
```
Comment (melissaquezada94) [inline core/designsystem/src/main/java/com/iteneum/designsystem/components/Text.kt:26]: Same comment here:
`I don't see the param onClick in the function, could you remove the param from the documentation if it is not need in the function or add the param onClick in the function if it need`

File: core/designsystem/src/main/java/com/iteneum/designsystem/components/Text.kt [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+@author Irving Ulises Gonzalez Uscanga
+
+[LPTitleSmall] is a composable function that displays a small-sized title text with the specified label, color, and optional text alignment. It can also be clickable and trigger an onClick event.
+@param modifier Modifier is used to apply custom styles and parameters to the title, such as the width and height of the component.
+@param label The text label to be displayed as the title.
+@param color The color of the title text.
+@param textAlign Optional text alignment parameter, default is TextAlign.Center.
+@param onClick Optional high-order function that defines the click behavior when the title is clicked.
```
Comment (melissaquezada94) [inline core/designsystem/src/main/java/com/iteneum/designsystem/components/Text.kt:34]: Same comment here:
`I don't see the param onClick in the function, could you remove the param from the documentation if it is not need in the function or add the param onClick in the function if it need`

File: core/designsystem/src/main/java/com/iteneum/designsystem/components/Text.kt [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+    modifier: Modifier = Modifier,
+    label: String,
+    color: Color,
+    textAlign: TextAlign? = TextAlign.Center
+) {
+    Text(
+        text = label,
+        color = color,
+        style = LPTypography.titleLarge,
+        modifier = modifier, textAlign = textAlign)
```
Comment (melissaquezada94) [inline core/designsystem/src/main/java/com/iteneum/designsystem/components/Text.kt]: Please move this code in the next line `textAlign = textAlign)` same for the next functions

**Review by melissaquezada94 — APPROVED** [review]

**Review by carlosph816 — APPROVED** [review]
Well done!

**Pull request merged**